gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    n = 5
</pre><p>Let us use more meaningfull names. n is n_samples? h is height?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>